### PR TITLE
shows out of stock instead of add to cart

### PR DESCRIPTION
**Pull request by Turkidev**

### CODE REVIEW

File: themes/default/views/product.blade.php [hunk 1]
```diff
@@ -37,28 +37,33 @@ class="@if ($category->name == $categoryItem->name) text-secondary-900 pl-3 !bor
                     <p>{{ $category->description }}</p>
                 </div>
                 <div class="grid grid-cols-3 gap-4 mt-4">
-                    @foreach ($category->products()->orderBy('order')->get() as $product)
```
Comment (CorwinDev) [inline themes/default/views/product.blade.php:40]: Please revert this change

File: themes/default/views/product.blade.php [hunk 2]
```diff
@@ -37,28 +37,33 @@ class="@if ($category->name == $categoryItem->name) text-secondary-900 pl-3 !bor
...
+                    @foreach ($category->products()->get() as $product)
                         <div class="md:col-span-1 col-span-3">
                             <div class="content-box h-full flex flex-col">
                                 <div class="flex gap-x-3 items-center mb-2">
-                                    @if ($product->image !== 'null')
-                                        <img src="{{ $product->image }}" alt="{{ $product->name }}"
-                                            class="w-14 rounded-md" onerror="removeElement(this);">
-                                    @endif
+                                    <img src="{{ $product->image }}" alt="{{ $product->name }}" class="w-14 rounded-md"
+                                        onerror="removeElement(this);">
```
Comment (CorwinDev) [inline themes/default/views/product.blade.php:45]: Also revert this change

File: themes/default/views/product.blade.php [hunk 3]
```diff
@@ -37,28 +37,33 @@ class="@if ($category->name == $categoryItem->name) text-secondary-900 pl-3 !bor
...
                                     <div>
                                         <h3 class="text-lg text-secondary-800 leading-5 font-semibold">
                                             {{ $product->name }}</h3>
                                         <p>{{ $product->price() ? config('settings::currency_sign') . $product->price() : __('Free') }}
                                         </p>
                                     </div>
                                 </div>
-                                <p>{!! Stevebauman\Purify\Facades\Purify::clean(Str::markdown(str_replace("\n", '<br>', $product->description))) !!}
+                                <p>{{ \Illuminate\Mail\Markdown::parse(str_replace("\n", '<br>', $product->description)) }}
```
Comment (CorwinDev) [inline themes/default/views/product.blade.php:53]: And this one as well

**Review by CorwinDev — CHANGES_REQUESTED** [review]

File: themes/default/views/product.blade.php (hunk 1 above)
Comment (Turkidev) [reply themes/default/views/product.blade.php:40]: it wont work if i didn't do it like this
it well always show out of stock

**Review by Turkidev — COMMENTED** [review]
out of stock
